try {
  // ... existing code ...
} catch (_error) {
  // ... existing code ...
}